tests/Tests/400__ods_dashboard/414__ods_dashboard_api.robot

```diff
@@ -776,15 +831,35 @@ Set Username In PVC Payload
     ${complete_pvc}=     Replace String    ${PVC_ENDPOINT_BODY}    <PVC_NAME>    ${username}
     [Return]    ${complete_pvc}
 
-Set Username In Notebook Payload
+Fill In Notebook Payload For Creation/Update
     [Documentation]     Fill in the json body for creating/updating a Notebook with the username
-    [Arguments]     ${notebook_username}
-    ${complete_pvc}=     Replace String    ${NB_ENDPOINT_BODY}    <NB_USERNAME>    ${notebook_username}
-    [Return]    ${complete_pvc}
+    [Arguments]     ${imagetagname}     ${notebook_username}=${NONE}
+    IF    "${notebook_username}" == "${NONE}"
+        ${body}=       Set Variable      ${NB_ENDPOINT_BODY_A}
+    ELSE
+        ${body}=       Replace String    ${NB_ENDPOINT_BODY_B}    <USERNAME>    ${notebook_username}
+    END
+    ${complete_body}=     Replace String    ${body}    <IMAGETAGNAME>    ${imagetagname}
+    [Return]    ${complete_body}
+
+Fill In Notebook Payload For Stopping
+    [Documentation]     Fill in the json body for creating/updating a Notebook with the username
+    [Arguments]     ${notebook_username}=${NONE}
+    IF    "${notebook_username}" == "${NONE}"
+        ${complete_body}=       Set Variable      ${NB_STOP_ENDPOINT_BODY_A}
+    ELSE
+        ${complete_body}=       Replace String    ${NB_STOP_ENDPOINT_BODY_B}    <USERNAME>    ${notebook_username}
+    END
+    [Return]    ${complete_body}
 
 Delete Test PVCs
```

Comment:
In tests/Resources/Page/ODH/JupyterHub/PVCChange.resource there is a keyword named `May Be Delete PVC` to delete one PVC. 

Would you mind (in this PR or another one) creating a new file OCPDashboard/Storage/Storage.robot and placing both keywords (could be renamed to `Delete PVC` and `Delete PVC List` or similar

Reply:
okay will do in another PR